src/symbols.c

```diff
@@ -2289,6 +2290,32 @@ static gint get_fold_header_after(ScintillaObject *sci, gint line)
 }
 
 
+/* returns the line after following all brace match for @brace on @line */
+static gint resolve_matching_braces(ScintillaObject *sci, gint line, gint brace)
+{
+	gint pos = sci_get_position_from_line(sci, line);
+	gint line_end = sci_get_line_end_position(sci, line);
+	gint lexer = sci_get_lexer(sci);
```

Comment:
`const` is good, `const` should be used more :-D

That said, nobody should sit down and add it to existing code, just use it on new code when you can.

Modifying existing code can be problematical, passing a `const` to existing functions with non-const parameters would cascade `const` changes everywhere through code that didn't need to be modified.  So its not worth it to change old code unless its local only.